ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Update-RsSubscription
+{
+  <#
+        .SYNOPSIS
+            This script will update subscriptions piped from get-RsSubscriptions
+
+        .DESCRIPTION
+            This script will take the custom object producted by get-RSSubscription and use the data to update the 
+            matchdata xml to either a new startdatetime, or enddate.
+
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+
+        .EXAMPLE
+            $proxy = "http://ssrs-t-1234/Reportserver"
+            Get-RsSubscription -ReportServerUri $proxy -path '/Finance/ImportantReports' | 
+            where description -eq 'super important' | Update-RsSubscription -ReportServerUri $proxy -EndDate 9/9/2099
+
+            assign proxy to variable for reuse.
+            use get-rsssubscripion to find the 'super important' report in my /finance/importantreports path. Then Pipe the results
+            to update-RsSubscription where we reuse the same $proxy to update the EndDate to 9/9/2099.
+
+        .EXAMPLE
+            $proxy = "http://ssrs-t-1234/Reportserver"
+
+            [psobject]$subscriptions = Get-RsSubscription -ReportServerUri $proxy -path '/Finance/ImportantReports' | where description -eq 'super important' 
+            $subscriptions | Update-RsSubscription -ReportServerUri $proxy -EndDate 9/9/2099 -StartDate 1/1/2016
+
+            assign proxy to variable for reuse. 
+            Then create a psobject variable to store the objects created from get-RSSubscription.
+            Then Pipe the results to Update-RSSubscription where we reuse the same $proxy to update the EndDate to 9/9/2999 and StartDate to 1/1/2016
+        
+    #>
+    
+  [CmdletBinding()]
+  param (
+    [string]
+    $ReportServerUri,
+    [Alias('ReportServerCredentials')]

```

Comment:
For new functions you don't need alias , those are only for backward compatibility in existing functions